test/services/conversion/TransitAccompanyingDocumentConversionServiceSpec.scala

```diff
@@ -46,15 +50,59 @@ class TransitAccompanyingDocumentConversionServiceSpec
     with ScalaFutures
     with IntegrationPatience {
 
-  private val countries = Seq(Country("valid", "AA", "Country A"), Country("valid", "BB", "Country B"))
+  private val countries                 = Seq(Country("valid", "AA", "Country A"), Country("valid", "BB", "Country B"))
+  private val kindsOfPackage            = Seq(KindOfPackage("P1", "Package 1"), KindOfPackage("P2", "Package 2"))
+  private val documentTypes             = Seq(DocumentType("T1", "Document 1", transportDocument = true), DocumentType("T2", "Document 2", transportDocument = false))
+  private val additionalInfo            = Seq(AdditionalInformation("I1", "Info 1"), AdditionalInformation("I2", "info 2"))
+  private val sensitiveGoodsInformation = Nil
 
   implicit private val hc: HeaderCarrier = HeaderCarrier()
 
   val validModel = models.TransitAccompanyingDocument(
-    localReferenceNumber = "lrn",
+    localReferenceNumber = "MRN GOES HERE",
     declarationType = DeclarationType.T1,
     countryOfDispatch = Some(countries.head.code),
-    countryOfDestination = Some(countries.head.code)
+    countryOfDestination = Some(countries.head.code),
+    transportIdentity = Some("identity"),
+    transportCountry = Some(countries.head.code),
+    numberOfItems = 1,
+    numberOfPackages = 3,
+    grossMass = 1.0,
+    principal =
+      models.Principal("Principal name", "Principal street", "Principal postCode", "Principal city", countries.head.code, Some("Principal EORI"), Some("tir")),
+    consignor = Some(models.Consignor("consignor name", "consignor street", "consignor postCode", "consignor city", countries.head.code, None, None)),
+    consignee = Some(models.Consignee("consignee name", "consignee street", "consignee postCode", "consignee city", countries.head.code, None, None)),
```

Comment:
Could this one be updated to test the presence of the consignee EORI instead of the name/address